README.md

```diff
@@ -1,3 +1,517 @@
 # React Native DateTimePicker
 
-[![CircleCI](https://circleci.com/gh/react-native-community/react-native-datetimepicker.svg?style=svg)](https://circleci.com/gh/react-native-community/react-native-datetimepicker)
+[![CircleCI Status](https://img.shields.io/circleci/project/github/react-native-community/react-native-datetimepicker/master.svg)](https://circleci.com/gh/react-native-community/workflows/react-native-datetimepicker/tree/master) ![Supports Android and iOS](https://img.shields.io/badge/platforms-android%20|%20ios-lightgrey.svg) ![MIT License](https://img.shields.io/npm/l/@react-native-community/slider.svg)
+
+React Native date & time picker component for iOS and Android
+
+## Getting started
+
+`npm install react-native-datetimepicker --save`
+
+or
+
+`yarn add react-native-datetimepicker`
+
+
+### Using React Native Link
```

Comment:
Perhaps this should read `### Install using react-native link`. Now it reads as if the goal is to use `link` it should be about adding the components to the native builds.